nanomq/apps/broker.c

```diff
@@ -474,16 +476,146 @@ broker(const char *url)
 #endif
 }
 
-int
-broker_start(int argc, char **argv)
+void *
+print_usage(void)
 {
+	fprintf(stderr, USAGE);
+}
+
+int status_check(pid_t *pid)
+{
+	char* data = NULL;
+	size_t size = 0;
+
 	int rc;
-	if (argc != 1) {
-		fprintf(stderr, "Usage: broker start <url>\n");
+	if ((rc = nng_file_get(PID_PATH_NAME, (void*)&data, &size)) != 0) {
```

Comment:
Error. should be
(((rc = nng_file_get(PID_PATH_NAME, (void*)&data, &size))) != 0)
Suggestion.
rc = nng_file_get(PID_PATH_NAME, (void*)&data, &size));
if (rc ==...

Reply:
Well, it is correct